entrypoint.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env sh
 
-set -o errexit
-
-IDEA_JDK="/app/idea-IC/jre64"
-export IDEA_JDK
-TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
-export TMPDIR
+export JAVA_HOME=/app/jdk
+export JAVA_TOOL_OPTIONS=-Djava.io.tmpdir=${XDG_CACHE_HOME}/tmp
```

Comment:
I think this is not needed, along with `"--filesystem=xdg-cache/tmp:create"`. There was an issue recently about setting `JAVA_TOOL_OPTIONS`. You should be getting a warning at startup because of this.

But regardless, it's not needed any more, is it? My understanding is that the JVM should be using `TMPDIR` by default. `java -XshowSettings` will tell you while using the `Terminal` from IntelliJ IDEA.

Reply:
Without that system property set, it will point to /tmp. I just tried and used java -XshowSettings. I think it's a good idea to make it point to a xdg cache directory, but if you think it can cause issues I can remove it.